.circleci/config.yml

```diff
@@ -86,6 +86,7 @@ jobs:
             pip install -e 'git+https://github.com/ckan/ckantoolkit.git#egg=ckantoolkit' -r 'https://raw.githubusercontent.com/ckan/ckantoolkit/master/requirements.txt'
             pip install -e 'git+https://github.com/open-data/frictionless-py.git@canada-v2.10#egg=frictionless' -r 'https://raw.githubusercontent.com/open-data/frictionless-py/canada-v2.10/requirements.txt'
             pip install -e 'git+https://github.com/open-data/ckanext-security.git@canada-v2.10#egg=ckanext-security' -r 'https://raw.githubusercontent.com/open-data/ckanext-security/canada-v2.10/requirements.txt'
+            pip install -e 'git+https://github.com/NorwegianRefugeeCouncil/ckanext-api-tracking.git#egg=ckanext-api-tracking' -r 'https://raw.githubusercontent.com/NorwegianRefugeeCouncil/ckanext-api-tracking/main/requirements.txt'
```

Comment:
likely want to fork this so we can control releases

Reply:
I did do a bunch of upstream contribs this week that were accepted. But added some more this morning and last night that would need to be brought in. So a fork might be needed, but I really don't want to Fork another repo hahaha

Reply:
what if they merge or change something that breaks your tidy release process though? I guess you could pin commit ids here..

Reply:
hmm yeah, okay I will fork the repo and make a v2.10 branch

Reply:
I always went with canada-XX for the branch names to not confuse with upstream releases

Reply:
forked and branched with my additional code changes for us. (they are all already in upstream PRs)